examples/gallery/images/track_sampling.py

```diff
@@ -25,18 +24,40 @@
 track = pygmt.grdtrack(points=points, grid=grid, newcolname="bathymetry")
 
 fig = pygmt.Figure()
-# Plot the earth relief grid on Cylindrical Stereographic projection, masking
-# land areas
+# Create global map using a Cylindrical Stereographic projection with a
+# width of 15 centimeters
 fig.basemap(region="g", projection="Cyl_stere/150/-20/15c", frame=True)
-fig.grdimage(grid=grid, cmap="gray")
-fig.coast(land="#666666")
-# Plot the sampled bathymetry points using circles (c) of 0.15 cm size
-# Points are colored using elevation values (normalized for visual purposes)
+
+# Set up a colormap for the elevation values of the Earth relief grid
+pygmt.makecpt(cmap="gray", series=[int(grid.min()), int(grid.max()), 10])
+# Plot the Earth relief grid with color-coding for the elevation
+fig.grdimage(grid=grid, cmap=True)
+# Add a colorbar for the elevation
+fig.colorbar(
+    position="JBC+o0c/1.2c+ml",  # Place colorbar at position Bottom Center
+    frame=["af", "x+lelevation", "y+lm"],
+)
+
+# Mask the land areas in gray and plot the shorelines in black with a
+# thickness of 0.5 points
+fig.coast(land="#666666", shorelines="1/0.5p,black")
+
+# Set up a colormap for the elevation values of the track points. These
+# values are normalized for visual purposes (see below)
+pygmt.makecpt(cmap="terra", series=[-1, 1, 0.01])
+# Plot the sampled bathymetry points using circles (c) with a diameter of
+# 0.15 centimeters (c). Points are colored using normalized elevation values
 fig.plot(
     x=track.longitude,
     y=track.latitude,
     style="c0.15c",
-    cmap="terra",
+    cmap=True,
     fill=(track.bathymetry - track.bathymetry.mean()) / track.bathymetry.std(),
 )
+# Add a colorbar for the normalized elevation
+fig.colorbar(
+    position="JRM+ml",  # Place colorbar at position Right Middle
+    frame=["a0.2f0.1", "x+lnormalized elevation"],
+)
+
```

Comment:
Usually I do like colorbars, but in this case, the normalized elevations are hard to interpret, so I'm not sure if adding a colorbar for it is helpful. If it were raw elevation values in metres, the colorbar would make sense, but then the colors of the points won't look nice :sweat_smile:

Reply:
> Usually I do like colorbars, but in this case, the normalized elevations are hard to interpret, so I'm not sure if adding a colorbar for it is helpful.

Yes, the numerical values are hard to interpret. But without a colorbar, one does not know to which quantity the color-coding referes and that the values were normalized.

> If it were raw elevation values in metres, the colorbar would make sense, but then the colors of the points won't look nice 😅

Yep, I tried this 😆.